test/fixtures/v2/documentRopstenObfuscatedWithEmptyObject.ts

```diff
@@ -0,0 +1,117 @@
+import { SchemaId, v2, WrappedDocument } from "@govtechsg/open-attestation";
+
+interface CustomDocument extends v2.OpenAttestationDocument {
+  recipient: any;
+  name: string;
+  description: string;
+  issuedOn: string;
+  admissionDate: string;
+  graduationDate: string;
+  transcript: any;
+  additionalData: any;
+  issuers: [
+    {
+      name: string;
+      url: string;
+      documentStore: string;
+      identityProof: any;
+    }
+  ];
+}
+export const documentRopstenObfuscatedWithEmptyObject: WrappedDocument<CustomDocument> = {
+  version: SchemaId.v2,
+  data: {
+    id: "c3eaaff0-be1b-44d6-a686-0dd313f3f74f:string:53b75bbe",
+    name: "b8275e90-35dc-47cc-b610-2bd1f5596c63:string:GovTech Demo Certificate",
+    description: "e5a63441-fa0c-407b-a4b4-2c12308844da:string:GovTech Demo Certificate",
+    issuedOn: "17cde367-5bdf-4e7d-b247-74556f631e37:string:2019-05-29T00:00:00+08:00",
+    admissionDate: "43d45748-e4d5-4310-8ae9-9500c6faa111:string:2017-08-01T00:00:00+08:00",
+    graduationDate: "218b3feb-d96d-4a0d-8ab9-e9510d0ccb64:string:2022-08-01T00:00:00+08:00",
+    $template: {
+      name: "cfe91df9-b118-4cdd-86b0-a8c9b5eaefe0:string:GOVTECH_DEMO",
+      type: "77ed5e5f-d427-4068-9821-20792ed9494f:string:EMBEDDED_RENDERER",
+      url: "201464fd-9971-4299-b317-76b49ee86b6e:string:https://demo-renderer.opencerts.io",
+    },
+    issuers: [
+      {
+        name: "1894d984-225c-434d-b923-38cb96213ea4:string:GovTech Singapore",
+        url: "09a3cf30-09ea-4ad5-9567-4110d0167620:string:https://tech.gov.sg",
+        documentStore: "6bf70b4b-6af6-41cc-8d7e-bbf47ded03cb:string:0x532C9Ff853CA54370D7492cD84040F9f8099f11B",
+        identityProof: {
+          type: "cb04ba9c-364e-4272-9b90-760842c0e66a:string:DNS-TXT",
+          location: "3376155f-b77f-477b-b50e-483574dee5e9:string:example.openattestation.com",
+        },
+      },
+    ],
+    recipient: {},
```

Comment:
I have two almost identical test certificate here. In this certificate, this is the result after obfuscating the last element (i.e. `recipient.name` etc. is obfuscated individually, rather than the entire `recipient` object).

From https://github.com/Open-Attestation/open-attestation/commit/a0c783ff399f0d8a3390dcf6173c4287a051082d#diff-af044c135a8496a28f0859e897dc301cR108, we expect such an obfuscated certificate to still be valid after `verify`ing it.

But somehow over here, `verify` returns saying it has been tampered. Not sure why though, am I missing something? 😨

Reply:
@Nebulis

Reply:
Nvm, I realise it was because we haven't upgraded `open-attestation`. I've done so in this MR